src/models/dynamic_generator.jl

```diff
@@ -5,9 +5,7 @@ mutable struct DynamicGenerator{
     TG <: TurbineGov,
     P <: PSS,
 } <: DynamicInjection
-    number::Int64
-    name::String
-    bus::Bus
+    static_injector::Union{Nothing, Generator}
     ω_ref::Float64
     V_ref::Float64
     P_ref::Float64
```

Comment:
We can remove P_ref and V_ref also. Those need to match the injector active power and bus Voltage respectively.